packages/console/src/monaco/MonacoProviders.tsx

```diff
@@ -272,6 +274,7 @@ class MonacoProviders extends PureComponent<
             title = `Fix ${d.code}`;
           }
         }
+        const b = 4;
```

Comment:
Seems to be left over testing code?

Reply:
Oops yes I put that there to see how vscode/ts service handles quick fixes